platformics/codegen/tests/test_field_visibility.py

```diff
@@ -74,12 +74,6 @@ async def test_hidden_mutations(
     output = await gql_client.query(query, user_id=user_id, member_projects=[project_id])
     mutations = [field["name"] for field in output["data"]["__schema"]["mutationType"]["fields"]]
 
-    # There are no mutable fields on GenomicRange (some are readonly, and others are mutable: false), so we shouldn't have a mutation for it.
-    assert "updateGenomicRange" not in mutations
-
-    # However we *can* create a genomic range.
-    assert "createGenomicRange" in mutations
```

Comment:
Since I added the `deletedAt` field to the base Entity class, and it's always a mutable field, every entity will now have an `update` mutation by default.